draft-ietf-mls-protocol.md

```diff
@@ -1638,6 +1621,11 @@ cross-group attacks.
     } MLSSignatureContent;
 ~~~
 
+The signature MUST cover the metadata information about the current state
+of the group (group identifier, epoch, generation and sender's Leaf index)
+to avoid Group participants to impersonate other participants and in order
+to prevent cross-group attacks.
```

Comment:
My point is that this wasn't normative. It's just descriptive. So the MUST should not be capitalized.

Reply:
Fixed, Thanks !